core/src/avm2/globals/flash/geom/Matrix3D.as

```diff
@@ -274,5 +274,8 @@ package flash.geom {
 			return new Matrix3D(this.rawData);
 		}
 
+		public function copyToMatrix3D(other:Matrix3D):void {
+			other.rawData = rawData
```

Comment:
Should this clone the data?

Reply:
It goes through the getter, so it automatically copies it.